filter-collection.json

```diff
@@ -0,0 +1,3295 @@
+{
+   "146d3286-8351-4318-b484-1f30e55f4d86":[
+      "universal-content"
+   ],
+   "a47bd115-5951-4834-b6b3-c302d1d532b7":[
+      "universal-content"
+   ],
+   "64c01fe0-d908-46d1-bd32-1a5714b74688":[
```

Comment:
change file format to UUIDs on new line and keep only UUIDS that are present in `wordpress` and `methode` collection